src/main/java/seedu/toluist/commons/util/StringUtil.java

```diff
@@ -55,4 +59,70 @@ public static String getDetails(Throwable t) {
     public static boolean isUnsignedInteger(String s) {
```

Comment:
so i rename it and still use it everywhere ah? since i dont want the list index to be 0 anyway